_includes/_troubleshooting/no-solution-available.html

```diff
@@ -0,0 +1 @@
+There is no clean solution for this. Please check the suggested workarounds below for further assistance.
```

Comment:
No verified solution available. Check the suggested [workaround](#workaround) instead.

If we choose this template phrasing, the section name should be Workaround (for the anchor to be working), even if it contains multiple workarounds.